src/Handler/Command.hs

```diff
@@ -64,9 +70,14 @@ getCommandR token = do
             $(widgetFile "command")
 
 deleteCommandR :: Token -> Handler ()
-deleteCommandR token = unsafeRunStorage $ do
-    del token
-    del $ History token
+deleteCommandR token = do
+    archived <- unsafeRunStorage $ do
```

Comment:
Heh, yes.

The (safe) `runStorage` returns an `Either` whereas the unsafe variant will raise any `Left` value with `error`. In the case of HTTP handlers, the default handling of `error` (500 + show the message) is perfectly fine with me for now.